metadata.yaml

```diff
@@ -11,7 +11,7 @@ owner: "ibmtfprovider"
 published_at: "2021-12-02T10:34:28.911362Z"
 registry: "https://registry.terraform.io/modules/terraform-ibm-modules/cluster-storage/ibm/latest"
 alias: "portworx"
-tags: ["portworx","storage","cluster","cluster-storage"]
+tags: ["portworx","storage","cluster","cluster-storage","odf"]
```

Comment:
portworx needs to removed here?

Reply:
Since `portworx` and `odf` are combined in the same module directory, shouldn't they be combined in the same `metadata.yaml` file?